config/routes.rb

```diff
@@ -1,6 +1,16 @@
 Rails.application.routes.draw do
+  get '/home', to: 'home#index', as: 'home'
+  
+  devise_for :users 
+  # controllers: { registrations: "users/registrations" }
```

Comment:
- [ ] Kindly remove commented code from your project. It makes code hard to read.